alpine/APKBUILD.in

```diff
@@ -33,6 +33,8 @@ _libdir=/usr/lib
 _user=frr
 
 build() {
+	export ABUILD_APK_INDEX_OPTS="--allow-untrusted"
```

Comment:
have we always allowed untrusted 'til now, just it was the default? or is this something new?

Reply:
We do already --allow-untrusted, but for `apk add`. Since 3.20 we need this for `apk index` too.